.gitattributes

```diff
@@ -0,0 +1,10 @@
+* text=auto
+
+/.github            export-ignore
+/docs               export-ignore
+/tests              export-ignore
```

Comment:
I would keep tests and docs: I used them sometimes when I don't have an internet connection (plane/travel). But great idea otherwise :)

Reply:
Thanks for the review, it's fixed!

Reply:
Well, those 2 folders are what contribute the most at increasing the size of the downloaded archive.

If you want to have access to them offline, there is a way: clone the repo.